src/components/Accordion.tsx

```diff
@@ -0,0 +1,100 @@
+import { useId } from "@react-aria/utils";
+import { Dispatch, ReactNode, SetStateAction, useEffect, useState } from "react";
+import { useFocusRing } from "react-aria";
+import { Css, Icon, Palette, useTestIds } from "src/index";
+
+export interface AccordionProps {
+  title: ReactNode;
+  children: ReactNode;
+  disabled?: boolean;
+  defaultExpanded?: boolean;
+  size?: AccordionSize;
+  /** Adds a top border (enabled by default) */
+  topBorder?: boolean;
+  /** Adds a bottom border (disabled by default) */
+  bottomBorder?: boolean;
+  /**
+   * Used by AccordionList
+   * Allows multiple accordions to be expanded simultaneously (enabled by default)
+   */
+  index?: number;
+  setExpandedIndex?: Dispatch<SetStateAction<number | undefined>>;
+}
+
+export function Accordion(props: AccordionProps) {
+  const {
+    title,
+    children,
+    size,
+    disabled = false,
+    defaultExpanded = false,
+    topBorder = true,
+    bottomBorder = false,
+    index,
+    setExpandedIndex,
+  } = props;
+  const testIds = useTestIds(props, "accordion");
+  const id = useId();
+  const [expanded, setExpanded] = useState(defaultExpanded);
+  const { isFocusVisible, focusProps } = useFocusRing();
+
+  useEffect(() => { setExpanded(defaultExpanded); }, [defaultExpanded])
+
+  return (
+    <div
+      {...testIds.container}
+      css={{
+        ...Css.bGray400.if(topBorder).bt.if(bottomBorder).bb.$,
+        ...(size ? Css.wPx(accordionSizes[size]).$ : {}),
+      }}
+    >
+      <button
+        {...testIds.title}
+        {...focusProps}
+        aria-controls={id}
+        aria-expanded={expanded}
+        disabled={disabled}
+        css={{
+          ...Css.df.jcsb.gap2.aic.w100.p2.baseEm.outline("none").addIn(":hover", Css.bgGray100.$).$,
+          ...(disabled && Css.gray500.$),
+          ...(isFocusVisible && Css.boxShadow(`inset 0 0 0 2px ${Palette.LightBlue700}`).$),
+        }}
+        onClick={() => {
+          setExpanded(!expanded);
+          if (setExpandedIndex) setExpandedIndex(index);
+        }}
+      >
+        <span>{title}</span>
+        <span
+          css={{
+            transition: "transform 250ms linear",
+            transform: expanded ? "rotate(180deg)" : "rotate(0deg)",
```

Comment:
**thought (non-blocking):** This is not a request at all, more-so for my own musing. If we could come up with a list of support transitions or transforms and some how bake them into Truss, that'd be pretty cool. Maybe something like:
```
Css.transition("transform").fast.rotate0.if(expanded).rotate180.$
```
"fast" could represent a predetermined duration, and we could also include ease-in or linear somehow? This will be a bigger convo with design to figure out some animation patterns.